src/main/java/jenkins/plugins/openstack/compute/OpenStackDSL.java

```diff
@@ -0,0 +1,26 @@
+package jenkins.plugins.openstack.compute;
+
+import groovy.lang.Binding;
+import hudson.Extension;
+import org.jenkinsci.plugins.workflow.cps.CpsScript;
+import org.jenkinsci.plugins.workflow.cps.GlobalVariable;
+
+@Extension public class OpenStackDSL extends GlobalVariable {
```

Comment:
I presume this is not an API for other plugins to use. Annotate with `@Restricted(NoExternalUse.class)`.